### PR TITLE
Dequeue Google Fonts

### PR DESCRIPTION
**Pull request by ShoboySnr**
This PR dequeue Google fonts

### CODE REVIEW

**Comment by ShoboySnr** [comment]
Okay

**Comment by ShoboySnr** [comment]
I have pushed @w3guy please check

**Comment by w3guy** [comment]
@ShoboySnr great job. Did you actually test and confirmed it works as expected?

**Comment by ShoboySnr** [comment]
Yes i did. When the dequeue  check box is checked, it removes the Google googleapis script and that Webfont function script.

File: src/Admin/SettingsPage/Settings.php [hunk 1]
```diff
@@ -112,6 +112,11 @@ public function settings_admin_page_callback()
                             'label'       => __('Disable Impression Tracking', 'mailoptin'),
                             'checkbox_label' => __('Disable', 'mailoptin')
                         ],
+                        'dequeue_google_font'     => [
+                            'type'        => 'checkbox',
+                            'label'       => __('Dequeue Google Fonts', 'mailoptin'),
```
Comment (w3guy) [inline src/Admin/SettingsPage/Settings.php:117]: @ShoboySnr please change to 
```suggestion
                            'label'       => __('Disable Google Fonts', 'mailoptin'),
```

File: src/Admin/SettingsPage/Settings.php [hunk 2]
```diff
@@ -112,6 +112,11 @@ public function settings_admin_page_callback()
...
+                            'checkbox_label' => __('Dequeue', 'mailoptin')
```
Comment (w3guy) [inline src/Admin/SettingsPage/Settings.php:118]: And this too.

```suggestion
                            'checkbox_label' => __('Disable', 'mailoptin')
```

**Review by w3guy — CHANGES_REQUESTED** [review]

**Pull request merged**